ui/app/AppLayouts/Chat/panels/GroupChatPanel.qml

```diff
@@ -76,24 +82,29 @@ RowLayout {
          for (var i = 0; i < groupUsersModelListView.count; i ++) {
              var entry = groupUsersModelListView.itemAtIndex(i)
 
-             // Add all group users different than me
-             if(!entry.isAdmin) {
-                 d.groupUsersModel.insert(d.groupUsersModel.count,
+             // Add all group users
+             d.groupUsersModel.insert(d.groupUsersModel.count,
                                       {"pubKey": entry.pubKey,
                                        "name": entry.name,
-                                       "icon": entry.icon})
-             }
+                                       "tagIcon": entry.isAdmin ? "crown" : "",
+                                       "isReadonly": entry.isAdmin})
          }
 
          // Build contactsModel type from model type (to fit with expected StatusTagSelector format
          for (var j = 0; j < contactsModelListView.count; j ++) {
              var entry2 = contactsModelListView.itemAtIndex(j)
              d.contactsModel.insert(d.contactsModel.count,
-                                {"pubKey": entry2.pubKey,
-                                 "displayName": entry2.name,
-                                 "icon": entry2.icon,
-                                 "isIdenticon": false,
-                                 "onlineStatus": false})
+                                    {"pubKey": entry2.pubKey,
```

Comment:
I know it's only matter of style, but there is no need to wrap js dict keys in "", imo it looks cleaner without quotation marks.
Other minor comment: insert could be replaced by append.

Reply:
done!